### ISSUE TITLE
USB device still active even after physically unplugged

### ISSUE THREAD

**Issue by Glandos** [systemd/systemd#7018]
### Submission type

 - Bug report

### systemd version the issue has been seen with

> 234

### Used distribution

> Debian testing

### In case of bug report: Expected behaviour you didn't see

> After physically unplugging my USB device, the following command should return as-is:
```
> systemctl status /dev/disk/by-label/wifiswitch
● dev-disk-by\x2dlabel-wifiswitch.device
   Loaded: loaded
   Active: inactive (dead)
```

This was the case in systemd v233. Downgrading resolves the issue.

### In case of bug report: Unexpected behaviour you saw

> After physically unplugging my USB device, the device is still active:
```
> systemctl status /dev/disk/by-label/wifiswitch
● dev-disk-by\x2dlabel-wifiswitch.device - U3_Cruzer_Micro wifiswitch
   Follow: unit currently follows state of sys-devices-pci0000:00-0000:00:1d.7-usb1-1\x2d7-1\x2d7:1.0-host7-target7:0:0-7:0:0:0-block-sdc-sdc1.device
   Loaded: loaded
   Active: active (plugged) since Fri 2017-10-06 22:06:49 CEST; 42s ago
   Device: /sys/devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1
```

### In case of bug report: Steps to reproduce the problem

> You can try to give a name to a FAT32 partition on a USB device (with msdos partition table), and try to play with plug and unplugged.

`systemctl status` should return `inactive` after unplugging the device.

**Comment by arvidjaar** [comment]
Run `udevadm monitor -ku`, unplug USB device. Did kernel generate events when unplugging? When in doubt, post output here.

**Comment by Glandos** [comment]
Here are the two outputs. The first is with systemd 233-10, using packages `libsystemd0_233-10_amd64.deb libudev1_233-10_amd64.deb systemd_233-10_amd64.deb systemd-sysv_233-10_amd64.deb udev_233-10_amd64.deb` and the second is with systemd 234-1 using `libsystemd0_234-1_amd64.deb libudev1_234-1_amd64.deb systemd_234-1_amd64.deb systemd-sysv_234-1_amd64.deb udev_234-1_amd64.deb`

Outputs seem to be nearly identical to me… rfkill events are normally generated by the device plugged and unplugged events, so they can be ignored in this comparison.

### systemd 233-10

```
# udevadm monitor --kernel --udev 
monitor will print the received events for:
UDEV - the event which udev sends out after rule processing
KERNEL - the kernel uevent

KERNEL[89911.210388] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
KERNEL[89911.210598] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
KERNEL[89911.213436] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
KERNEL[89911.213584] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
UDEV  [89911.219263] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
UDEV  [89911.224255] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
UDEV  [89911.225987] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
UDEV  [89911.227469] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
KERNEL[89912.223321] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89912.227406] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89912.229002] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
KERNEL[89912.229108] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
KERNEL[89912.229201] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
KERNEL[89912.229324] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
KERNEL[89912.229492] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
KERNEL[89912.230116] add      /devices/virtual/bdi/11:0 (bdi)
KERNEL[89912.230873] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
KERNEL[89912.232596] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
UDEV  [89912.233109] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89912.233630] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
KERNEL[89912.234179] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
KERNEL[89912.234594] change   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89912.249678] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89912.250045] add      /devices/virtual/bdi/8:32 (bdi)
UDEV  [89912.250251] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
UDEV  [89912.250435] add      /devices/virtual/bdi/11:0 (bdi)
UDEV  [89912.250606] add      /devices/virtual/bdi/8:32 (bdi)
UDEV  [89912.250807] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
UDEV  [89912.251012] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
UDEV  [89912.251960] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
UDEV  [89912.255375] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
UDEV  [89912.258206] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
UDEV  [89912.258560] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
UDEV  [89912.262126] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
KERNEL[89912.269505] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
KERNEL[89912.269759] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
KERNEL[89912.269990] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
UDEV  [89912.317342] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89912.366945] change   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89912.475178] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
UDEV  [89912.581482] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
UDEV  [89912.683213] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
KERNEL[89919.973987] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
KERNEL[89919.974227] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
KERNEL[89919.974383] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
KERNEL[89919.974539] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
KERNEL[89919.975974] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
KERNEL[89919.977042] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
KERNEL[89919.977527] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
KERNEL[89919.978097] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
UDEV  [89919.979778] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
UDEV  [89919.980774] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
UDEV  [89919.981540] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
UDEV  [89919.981738] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
UDEV  [89919.984725] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
UDEV  [89919.985173] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
UDEV  [89919.986956] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
UDEV  [89919.987895] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89919.988696] remove   /devices/virtual/bdi/8:32 (bdi)
UDEV  [89919.993133] remove   /devices/virtual/bdi/8:32 (bdi)
KERNEL[89920.036809] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
KERNEL[89920.036999] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
KERNEL[89920.037105] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
KERNEL[89920.037676] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
KERNEL[89920.037945] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
UDEV  [89920.040289] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
UDEV  [89920.041882] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
UDEV  [89920.042719] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
UDEV  [89920.044900] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89920.045768] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
KERNEL[89920.052483] remove   /devices/virtual/bdi/11:0 (bdi)
KERNEL[89920.052710] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
UDEV  [89920.054766] remove   /devices/virtual/bdi/11:0 (bdi)
UDEV  [89920.055945] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89920.094070] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
KERNEL[89920.094256] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
KERNEL[89920.094430] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
KERNEL[89920.095736] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
UDEV  [89920.098271] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
UDEV  [89920.099539] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
UDEV  [89920.101306] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
UDEV  [89920.102832] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
KERNEL[89920.120275] change   /devices/pci0000:00/0000:00:1e.0/0000:05:01.0/ieee80211/phy0/rfkill0 (rfkill)
UDEV  [89920.124831] change   /devices/pci0000:00/0000:00:1e.0/0000:05:01.0/ieee80211/phy0/rfkill0 (rfkill)
^C⏎
```

### systemd 234-1

```
# udevadm monitor --kernel --udev 
monitor will print the received events for:
UDEV - the event which udev sends out after rule processing
KERNEL - the kernel uevent

KERNEL[89709.174571] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
KERNEL[89709.176494] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
KERNEL[89709.178519] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
KERNEL[89709.178992] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
UDEV  [89709.183523] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
UDEV  [89709.186640] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
UDEV  [89709.188424] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
UDEV  [89709.190226] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
KERNEL[89710.210073] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89710.214069] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89710.214507] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
KERNEL[89710.214623] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
KERNEL[89710.216870] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
KERNEL[89710.225381] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
UDEV  [89710.228938] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89710.229978] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
UDEV  [89710.231254] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89710.233086] add      /devices/virtual/bdi/11:0 (bdi)
UDEV  [89710.234050] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
KERNEL[89710.240909] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89710.242375] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
KERNEL[89710.243291] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
UDEV  [89710.252571] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
KERNEL[89710.253386] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
UDEV  [89710.254124] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
KERNEL[89710.254855] add      /devices/virtual/bdi/8:32 (bdi)
UDEV  [89710.255645] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
KERNEL[89710.256531] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
UDEV  [89710.257289] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
KERNEL[89710.258037] change   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
KERNEL[89710.258761] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
KERNEL[89710.259533] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
KERNEL[89710.260567] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
UDEV  [89710.266642] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
UDEV  [89710.267500] add      /devices/virtual/bdi/8:32 (bdi)
UDEV  [89710.271444] add      /devices/virtual/bdi/11:0 (bdi)
UDEV  [89710.275854] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
UDEV  [89710.332480] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89710.390724] change   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89710.499105] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
UDEV  [89710.675230] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
UDEV  [89710.706885] add      /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
KERNEL[89710.720385] change   /devices/pci0000:00/0000:00:1e.0/0000:05:01.0/ieee80211/phy0/rfkill0 (rfkill)
UDEV  [89710.727492] change   /devices/pci0000:00/0000:00:1e.0/0000:05:01.0/ieee80211/phy0/rfkill0 (rfkill)
KERNEL[89718.150792] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
KERNEL[89718.150942] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
KERNEL[89718.151052] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
KERNEL[89718.151274] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
KERNEL[89718.151647] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
KERNEL[89718.151935] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
KERNEL[89718.152567] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
KERNEL[89718.152715] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
UDEV  [89718.155061] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/bsg/7:0:0:0 (bsg)
UDEV  [89718.156395] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_generic/sg2 (scsi_generic)
UDEV  [89718.156949] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_device/7:0:0:0 (scsi_device)
UDEV  [89718.158254] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/scsi_disk/7:0:0:0 (scsi_disk)
UDEV  [89718.160595] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc1 (block)
UDEV  [89718.160991] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc/sdc2 (block)
UDEV  [89718.164786] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0/block/sdc (block)
UDEV  [89718.165795] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:0 (scsi)
KERNEL[89718.172509] remove   /devices/virtual/bdi/8:32 (bdi)
UDEV  [89718.174699] remove   /devices/virtual/bdi/8:32 (bdi)
KERNEL[89718.216581] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
KERNEL[89718.216823] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
KERNEL[89718.216928] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
KERNEL[89718.217502] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
KERNEL[89718.218029] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
UDEV  [89718.220375] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/bsg/7:0:0:1 (bsg)
UDEV  [89718.222733] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_generic/sg3 (scsi_generic)
UDEV  [89718.222983] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/scsi_device/7:0:0:1 (scsi_device)
UDEV  [89718.224715] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1/block/sr0 (block)
UDEV  [89718.225686] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0/7:0:0:1 (scsi)
KERNEL[89718.236459] remove   /devices/virtual/bdi/11:0 (bdi)
KERNEL[89718.236688] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
UDEV  [89718.238989] remove   /devices/virtual/bdi/11:0 (bdi)
UDEV  [89718.240267] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/target7:0:0 (scsi)
KERNEL[89718.268505] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
KERNEL[89718.268758] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
KERNEL[89718.269416] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
KERNEL[89718.269979] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)
UDEV  [89718.271970] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7/scsi_host/host7 (scsi_host)
UDEV  [89718.272894] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0/host7 (scsi)
UDEV  [89718.274004] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7/1-7:1.0 (usb)
UDEV  [89718.275238] remove   /devices/pci0000:00/0000:00:1d.7/usb1/1-7 (usb)

^C⏎
```

**Comment by poettering** [comment]
My educated guess is that this is fixed by cc0df6cc35339976c367977dc292278a1939db0c. Please update to 235 hence.

Closing this hence. If you manage to reproduce this even with 235, please reopen!

**Comment by Glandos** [comment]
You seems to have received a good education then.

It works with systemd 235.

I should have tried before (this version hit Debian/testing 7 days ago), but my laziness was too strong then. Thanks for waking me up on this.
